gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -163,6 +178,14 @@ def main(args):
         ht = vep_or_lookup_vep(ht, vep_version=args.vep_version)
         ht.write(res.vep_ht.path, overwrite=args.overwrite)
 
+    if args.validate_vep:
+        res = resources.validate_vep
+        res.check_resource_existence()
+        vep_ht = res.vep_ht.ht()
+        interval_ht = ensembl_interval.ht()
+        count_ht = count_vep_annotated_variants_per_interval(vep_ht, interval_ht)
```

Comment:
Actually now that I look at it, this can just be
```suggestion
        count_ht = count_vep_annotated_variants_per_interval(res.vep_ht.ht(), ensembl_interval.ht())
```